### PR TITLE
[Merged by Bors] - feat: add lemmas `nullMeasurableSet_lt'` and `nullMeasurableSet_le`

### PR DESCRIPTION
**Pull request by ocfnash**
Prior to this commit the state of Mathlib was:
```lean
import Mathlib

#check measurableSet_lt  -- Exists
#check measurableSet_lt' -- Exists
#check measurableSet_le  -- Exists

#check nullMeasurableSet_lt  -- Exists
#check nullMeasurableSet_lt' -- Missing
#check nullMeasurableSet_le  -- Missing
```

Co-authored-by: teorth <tao@math.ucla.edu>

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean [hunk 1]
```diff
@@ -643,6 +643,15 @@ theorem nullMeasurableSet_lt [SecondCountableTopology α] {μ : Measure δ} {f g
   (hf.prod_mk hg).nullMeasurable measurableSet_lt'
 #align null_measurable_set_lt nullMeasurableSet_lt
 
+theorem nullMeasurableSet_lt' [SecondCountableTopology α] {μ : MeasureTheory.Measure (α × α)} :
```
Comment (RemyDegenne) [inline Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean:646]:
```suggestion
theorem nullMeasurableSet_lt' [SecondCountableTopology α] {μ : Measure (α × α)} :
```

File: Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean [hunk 2]
```diff
@@ -643,6 +643,15 @@ theorem nullMeasurableSet_lt [SecondCountableTopology α] {μ : Measure δ} {f g
...
+    NullMeasurableSet { p : α × α | p.1 < p.2 } μ :=
+  measurableSet_lt'.nullMeasurableSet
+
+theorem nullMeasurableSet_le [SecondCountableTopology α] {μ : MeasureTheory.Measure δ}
```
Comment (RemyDegenne) [inline Mathlib/MeasureTheory/Constructions/BorelSpace/Basic.lean:650]:
```suggestion
theorem nullMeasurableSet_le [SecondCountableTopology α] {μ : Measure δ}
```

**Review by RemyDegenne — APPROVED** [review]
Thanks!
bors d+

**Comment by ocfnash** [comment]
bors merge

**Pull request closed without merging**